src/tty-ask-password-agent/tty-ask-password-agent.c

```diff
@@ -518,6 +566,14 @@ static int parse_argv(int argc, char *argv[]) {
                         arg_action = ACTION_QUERY;
                         break;
 
+              case ARG_FILTER:
+                        r = parse_filter(optarg ?: argv[optind]);
```

Comment:
conditionalization on optarg should be removed, the argument is not optional

Reply:
not even an isempty check ?

Reply:
parse_pid error handling is already doing that for us